src/data/en/index.ts

```diff
@@ -354,18 +358,23 @@ export const instanceData = {
         1060001: 'You successfully recovered your account. Please change your password in the next minutes.',
         1060002: 'An email containing a recovery link has been sent to the email address you provided.',
         1080001: 'An email containing a verification link has been sent to the email address you provided.',
-        1080002: 'You successfully verified your email address.',
+        1080002: emailVerifiedSuccessfully,
         4000001: '%reason%',
-        4000002: 'Property %field% is missing.',
+        4000002: '%field% is missing.',
         4000003: 'Length must be \u003e= %expected_length%, but got %actual_length%.',
         4000004: '"%value%" is not valid "%format%"',
         4000005: 'The password can not be used because %reason%.',
-        4000006: 'The provided credentials are invalid, check for spelling mistakes in your password or username/email.',
+        4000006: 'The username, email adress or password was wrong. Please check for spelling mistakes.',
         4000007: 'An account with the same email or username exists already.',
         4000008: 'The provided authentication code is invalid, please try again.',
-        4000010: 'Account not active yet. Did you already verify your email address?',
+        4000010: 'Have you already verified your email address?. %verificationLink%',
         4060004: 'The recovery token is invalid or has already been used. Please retry the flow.',
-      }
+      },
+      usernameRules: 'Your username may only contain letters, digits, underscores (_) and hyphens (-).',
+      registrationAgreement: 'By clicking %signup%, you agree to our %privacypolicy% and %terms%. You may receive email notifications from us and can opt out at any time.',
+      terms: 'Terms',
+      signUp: 'Sign up',
+      verificationLink: 'Go here to get the verification email again.'
```

Comment:
Maybe "Click to send verification email again"?

Reply:
We can do so. 
Just to make it clear: after clicking there you go to another page, where you have to insert the email.

Reply:
Hmm, I see. Then maybe smth like "Click here to request the verification email again"?